spec/features/people_spec.rb

```diff
@@ -199,15 +199,15 @@
     visit responseobj.invitation_set.invitation_url(invtoken.token_plain, false)
   end
 
-  it 'should unsubscribe when you click the unsubscribe button' do
+  xit 'should unsubscribe when you click the unsubscribe button' do
```

Comment:
Waarom is deze geskipped?

Reply:
Omdat die box maar tot 1 oktober wordt getoond en nu dus niet meer getoond wordt. Ik dacht dat we de logica er misschien nog wel in wilden laten staan om het elders opnieuw te gebruiken, en dat we dan ook de specs erbij houden. En een spec pending maken is een goede manier om die niet uit het oog te verliezen lijkt me (we kunnen m ook commenten als je het irritant vindt).